apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/context/case-generation-context/case-generation.context.ts

```diff
@@ -0,0 +1,11 @@
+import { CaseGenerationContext } from '@/pages/Entities/components/CaseGeneration/context/case-generation-context/types';
+import { createContext } from 'react';
+
+export const initialCaseGenerationContext = {
+  isMultipleCasesCreation: false,
+  isOpen: false,
+} as CaseGenerationContext;
```

Comment:
Prefer `satisfies` or `initialCaseGenerationContext: CaseGenerationContext` for this use.
```suggestion
} satisfies CaseGenerationContext;
```

Reply:
Won't work here. Initial context missing methods.